schema/deploy/trigger_functions/checksum_form_results.sql

```diff
@@ -29,7 +29,7 @@ begin
                                           and fr.version_number = new.version_number - 1)));
 
       if new_form_result_hash != old_form_result_hash then
-      raise notice 'Form Result: % changed from previous version', (select name from ggircs_portal.form_json where id = temp_row.form_id);
+        raise notice 'Form Result: % changed from previous version', (select name from ggircs_portal.form_json where id = temp_row.form_id);
```

Comment:
These notices are polluting the test output. Are they necessary?

Reply:
nope